### ISSUE TITLE
Cannot run geometry editor on systems without OpenGL 2.0

### ISSUE THREAD

**Issue by macumber** [openstudiocoalition/OpenStudioApplication#358]
Issue overview
--------------
Geometry Editor fails to run on systems without OpenGL 2.0

## Current Behavior
App will either crash or (on VirtualBox VMs) graphics will display inverted when going to geometry tab

## Expected Behavior
Geometry tab should work

## Steps to Reproduce
Run OpenStudio in Windows VM on VirtualBox with enable 3D graphics option off

## Possible Solution
Ship Angle OpenGL dlls (d3dcompiler_47.dll, libEGL.dll, libGLESv2.dll) with application

**Comment by jmarrec** [comment]
Windows server vm, launching the app I see this

```
User@WinDev2101Eval MINGW64 /
$ OPENSUDIO_APPLICATION_DEBUG=1 /c/openstudioapplication-1.2.0/bin/OpenStudioApp.exe
Critical: Failed to load libEGL (Unknown error 0x000000c1.)
Critical: Failed to load opengl32sw.dll (The specified module could not be found.)
```

When switching to geometry tab

![image](https://user-images.githubusercontent.com/5479063/120298904-c3daa280-c2ca-11eb-87d7-7c6e656ff819.png)

**Comment by jmarrec** [comment]
![image](https://user-images.githubusercontent.com/5479063/120298951-d0f79180-c2ca-11eb-8768-e101d10a3ff3.png)

**Comment by jmarrec** [comment]
```
julien@~$ cd /c/Qt/5.15.0/
julien@5.15.0$ find . -name "*.dll"
./msvc2019_64/bin/d3dcompiler_47.dll
./msvc2019_64/bin/libEGL.dll
./msvc2019_64/bin/libEGLd.dll
./msvc2019_64/bin/libGLESv2.dll
./msvc2019_64/bin/libGLESv2d.dll
[...]
````

**Comment by macumber** [comment]
As an interim fix, extract the zip file below and add the dlls to `C:\openstudioapplication-1.2.0\bin`:

[qt_opengl_dlls.zip](https://github.com/openstudiocoalition/OpenStudioApplication/files/6621126/qt_opengl_dlls.zip)

**Comment by jmarrec** [comment]
This is back on 1.4.0-rc1 with the move to Qt6. #418 had fixed it by putting 

```
    install(PROGRAMS $<TARGET_FILE:Qt5::OpenGL> DESTINATION bin COMPONENT OpenStudioApp )
    install(PROGRAMS $<TARGET_FILE:Qt5::Gui_EGL> DESTINATION bin COMPONENT OpenStudioApp )
    install(PROGRAMS $<TARGET_FILE:Qt5::Gui_GLESv2> DESTINATION bin COMPONENT OpenStudioApp )
```

That got removed in #461: https://github.com/openstudiocoalition/OpenStudioApplication/pull/461/files#diff-3e0e7d9040c69c7312cc9765c646281ec373ef0b11a098c8863e33216d629e3fR349

**Comment by macumber** [comment]
Is this OpenGL issue one that you want me to look at @jmarrec ?

**Comment by jmarrec** [comment]
There's also a `C:\Qt\6.3.0\msvc2019_64\lib\cmake\Qt6\FindGLESv2.cmake` which adds a `GLESv2::GLESv2` target.

**Comment by macumber** [comment]
We are also missing `dbghelp.dll` which was shipped with 1.3.1

**Comment by macumber** [comment]
[opengl32sw.qt-6.3.0.zip](https://github.com/openstudiocoalition/OpenStudioApplication/files/8674891/opengl32sw.qt-6.3.0.zip)
